numpy/add_newdocs.py

```diff
@@ -463,6 +463,46 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """))
 
+add_newdoc('numpy.core', 'nested_iters',
+    """
+    Create a ordered sequence of iterators. They are nested so that iterating
+    one will reset all the iterators after it in the sequence.
+
+    Parameters 
+    ----------
+    op : ndarray
+        The array to iterate over.
+
+    axes :  tuple of tuples
```

Comment:
Extra space. Sorry for not catching this last time, but should be `tuple of tuple of int`, or simlar